src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: External services setup
+metaDescription: "Here are steps to enable and configure external services"
+---
+The external services feature is available for New Relic APM agents and OpenTelemetry. After you follow the steps below to set up external services, you may need to adjust the [sampling rate](#adjust-sampling) if you don't see enough data.
+
+## APM agents [#externals-and-apm]
+
+These steps are primarily for the expanded external services feature that is based on distributed tracing. 
+
+<Callout variant="tip">
+The classic version of external services is still available if you need to add a new service to a chain of agents already using classic external services. To install classic external services, you enable [cross-application tracing](/docs/apm/transactions/cross-application-traces/introduction-cross-application-traces) instead of distributed tracing in the steps below.
```

Comment:
same comment re: dash in CAT

Reply:
@barbnewrelic, yikes, my reflex to hyphenate is strong!